### DISCUSSION TITLE
[Feature Request] Update GO from within the program

### DISCUSSION THREAD

**Discussion by vpoguru** [GrandOrgue/grandorgue#1554] [General]
Is there a possibility to implement the option to update GO from within the program in the future?

**Comment by oleg68** [comment]
There is an open issue #704 for it.

I'm not going to implement the software update from within GO, but I'm going to implement check for a new version available.

**Comment by kerkovits** [comment]
Linux package managers can auto-update from repositories. GrandOrgue is available at flathub, AUR, opensuse, and copr. Unfortunately, the Ubuntu repository is outdated.

**Comment by lbssousa** [comment]
Related issue for Windows: https://github.com/GrandOrgue/grandorgue/issues/1549
